source/computational_domain.cc

```diff
@@ -589,9 +599,8 @@ void ComputationalDomain<dim>::refine_and_resize(const unsigned int refinement_l
       pcout<<"Used tolerance is: "<<tolerance<<endl;
       for (unsigned int i=0; i<cad_surfaces.size(); ++i)
         {
-          OpenCASCADE::NormalToMeshProjectionBoundary<2,3> normal_to_mesh_projector(cad_surfaces[i], tolerance);
-
-          normal_to_mesh_projectors.push_back(normal_to_mesh_projector);
+          normal_to_mesh_projectors.push_back(std::shared_ptr<OpenCASCADE::NormalToMeshProjectionBoundary<2,3> >
+                                    (new OpenCASCADE::NormalToMeshProjectionBoundary<2,3>(cad_surfaces[i], tolerance)));
```

Comment:
SP is not recognized by the compiler. What should I include to use that function?